components/calendar/src/provider.rs

```diff
@@ -190,5 +376,104 @@ pub struct WeekDataV2 {
     pub min_week_days: u8,
     /// Bitset representing weekdays that are part of the 'weekend', for calendar purposes.
     /// The number of days can be different between locales, and may not be contiguous.
-    pub weekend: WeekendSet,
+    weekend_set: WeekendSet,
```

Comment:
also needs custom `Bake`

Reply:
Addressed in https://github.com/unicode-org/icu4x/pull/4364/commits/2433437fffd434b4317da42134e15cb1a01ca81c

Reply:
Actually, why is this field private?

Reply:
Because of https://github.com/unicode-org/icu4x/pull/4364#discussion_r1405427303

Reply:
The `u8` inside `WeekendSet` has to be private, because it has the invariant that the 8th bit is zero. The type here however does not have any invariants so `weekend_set` can be a public field.